index.html

```diff
@@ -9,49 +9,42 @@
 
 <body> 
 
-  <section class="navigation">
-    <div class="toolbar-container">
-        <div class="logo">Welcome</div>
+  <header class="navigation">
+    <nav class="toolbar-container">
+        <div class="logo"><a href="#welcome">Welcome</a></div>
         <div class="menu-bar">
           <a href="#portoflio">Portfolio</a>
           <a href="#about">About</a>
           <a href="#contact">Contact</a>
         </div> 
-        <a class="icon" >
-            <i class="fa fa-bars"></i>
-      </div>
-    </section>
-    <br>
-    <br>
-    
-    
-    <section class="headline">
+        <div>
+        <a class="icon" ><a href="#menu-bar">
+            <i class="fa fa-bars"></i></a>
+        </div>
+    </nav>
+  </header> 
+  <section class="headline back-image">
       
       <div class="primary-headline">
             <h1>Hey there. I'm Hiwot<span class="h1color"><br>
-             I'm software developer</span>
+             I'm a software developer</span>
             </h1>
-       
-        
+      </div>
+      <div>
             <p class="supporting-text">  I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
                 don’t hestiate to contact me.
             </p>
-        
-
-        <div class="social">
+        </div>
+        <div>
         <ul class="socialt">
             <li>  <img src="image/twitter.svg" /> &nbsp;</li>
             <li> <img src="image/vector.png" /> </li>
             <li> <img src="image/vector (1).png" />&nbsp; </li>
             <li> <img src="image/medium.svg" /> </li>&nbsp;
             <li> <img src="image/github.svg" /> &nbsp;</l>
-            <li> <img src="image/angellist.svg" /> &nbsp;</l>
-            
+            <li> <img src="image/angellist.svg" /> &nbsp;</l>   
         </ul> 
```

Comment:
- [ ] as requested by the previous reviewer, please wrap your social icons in  an anchor tag so that a usual user can use the icons to connect with you on various social platforms. 

example : 
            `<li> <a href='#' > <img src="image/vector.png" /> </a> </li>`